packages/deprecated/foundation-composable/src/Composable.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
+
 import * as React from 'react';
```

Comment:
Why did we have to ban these?

Reply:
Or is it just because all of these are in the `deprecated` path so we don't quite care to lint them?

Reply:
These use `object` extensively. Instead of adding `// eslint-disable-next-line @typescript-eslint/ban-types` everywhere, I opted to disable entirely since they are deprecated.